### ISSUE TITLE
Currently playing on dir.xiph.org (YP) URL-escaped

### ISSUE THREAD

**Issue by Moonbase59** [AzuraCast/AzuraCast#6240]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #4b05857 (2023-04-24 15:58)

### What happened?

When adding your station to the Xiph.org Yellow Pages ([dir.xiph.org](https://dir.xiph.org/)), the yellow pages show an URL-escaped string for the title currently playing:

```
On Air: Inkubus%20Sukkubus%20%2d%20The%20Rape%20of%20Maude%20Bowen
```

This is the same for @LordHelmchen666’s Eisradio, btw, and other (non-AzuraCast) stations are shown correctly.

Wonder if this is an AzuraCast or upstream (icecast-kh) issue?

**Comment by BusterNeece** [comment]
Definitely an upstream issue with Icecast-KH. We have nothing to do with how YP data gets sent over.

**Comment by Moonbase59** [comment]
Thanks for the feedback. Created an upstream issue at https://github.com/karlheyes/icecast-kh/issues/418.

**Comment by Moonbase59** [comment]
Fixed upstream, in Icecast 2.4.0-kh21.3, build 2-g5f5564d, and confirmed working with dir.xiph.org and internet-radio.com.
